packages/grafana-ui/src/components/Select/DropdownIndicator.tsx

```diff
@@ -6,6 +6,7 @@ interface DropdownIndicatorProps {
 }
 
 export const DropdownIndicator: React.FC<DropdownIndicatorProps> = ({ isOpen }) => {
-  const icon = isOpen ? 'angle-up' : 'angle-down';
-  return <Icon name={icon} />;
+  const icon = isOpen ? 'search' : 'angle-down';
```

Comment:
Can we define a `searchingIcon` or `typingIcon` property which is defaulted to "search"?

Reply:
not sure that is a good idea. Think having different icons in different selects would be confusing

Reply:
or maybe I misunderstood you, what did you mean?